tests/test-cases/Group1-Docker-Commands/1-06-Docker-Run.robot

```diff
@@ -65,14 +65,14 @@ Docker run fake image
     Should Contain  ${output}  "fakeImage" is not a valid repository/tag
```

Comment:
Commenting here since I can't seem to add a comment against line 56:
```
docker %{VCH-PARAMS} run busybox fakeCommand
```
should use `${busybox}`